cime_config/acme/config_archive.xml

```diff
@@ -132,33 +134,33 @@
   <comp_archive_spec compname="dwav" compclass="wav">
     <rest_file_extension>\.r.*</rest_file_extension>
     <rest_history_varname>unset</rest_history_varname>
-    <rpointer_file>rpointer$NINST_STRING.wav</rpointer_file>
```

Comment:
I didn't see anything that would adversely affect acme - I think that these are previous updates made on the cesm side that were not made on the acme side.   When acme customizes the config_archive.xml for external acme components we will need to take care not to overwrite them, but afaict that hasn't happened yet.